pom.xml

```diff
@@ -2090,7 +2107,7 @@
       </build>
     </profile>
     <profile>
-      <id>pitest-checkstyle-api-filters</id>
+      <id>pitest-checkstyle-api</id>
```

Comment:
https://github.com/checkstyle/checkstyle/blob/master/shippable.yml
This file must be updated if you are changing pitest.